### DISCUSSION TITLE
training chinese tts model

### DISCUSSION THREAD

**Discussion by arthurwu4work** [coqui-ai/TTS#2575] [General Q&A]
Hello,

I want to training a voice clone model (YourTTS) using aishell3 dataset.
Do I need to convert the transcript to phoneme sequence first ?
ex : audio1 | 鼻子 --> audio1 | B IY2 Z IY5 

Do I need to add a period to the end of the text?
ex : audio1 | 鼻子 --> audio1 | 鼻子。

Are there any points to note during the model training?

Thank you

**Comment by HuJunPariSmile** [comment]
i use audio1 | 鼻子 --> audio1 | 鼻子 is now woking
